kubernetes/collector.yaml

```diff
@@ -30,10 +30,11 @@ spec:
             - NET_RAW
             - NET_ADMIN
         env:
-        #- name: NETSIL_SERVICE_HOST
-        #  value: '<AOC address here>'
-        #- name: NETSIL_ORGANIZATION_ID
-        #  value: '<Netsil Organization ID>'
+        # Please DO NOT prepend http:// or https:// to the NETSIL_SERVICE_HOST value
```

Comment:
Are we intentionally uncommenting these two values?

Reply:
missed your review.. the docs show these values as uncommented, but also says to "uncomment" them. Was trying to make the files we provide consistent with the files the docs say to create, but maybe it is better to change the docs?

Reply:
I think the docs only say to uncomment them in the pre-1.6 kubernetes manifests.
In a sense, this `collector.yaml` is not really used now, since we no longer link to it from the docs. In fact, the manifest is templated and displayed in-documentation now.

I'd say let's recomment it here, so it's at least consistent with the pre-1.6 manifest that's also in this repo (and which we do link from the docs), and so it's not a manifest with "incorrect" values.